client/blocks/reader-site-notification-settings/index.jsx

```diff
@@ -122,19 +122,20 @@ class ReaderSiteNotificationSettings extends Component {
 					className="reader-site-notification-settings__popout"
 				>
 					<div className="reader-site-notification-settings__popout-toggle">
-						{ translate( 'New post notifications' ) }
+						{ translate( 'Notify me of new posts' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **66** times:
`translate( 'Notify me of new posts via email.' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).